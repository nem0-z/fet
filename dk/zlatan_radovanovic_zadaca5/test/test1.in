a = 56*([5 + 4.7 * (6-2/3) - 8.875] + 5.87)
print a
