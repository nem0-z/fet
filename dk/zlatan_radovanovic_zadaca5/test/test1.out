1515.453333
